dev-lang/icon/icon-9.5.20h.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="very high level language"
+HOMEPAGE="http://www.cs.arizona.edu/icon/"
+
+SRC_URI="https://github.com/gtownsend/icon/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain HPND"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="X iplsrc"
+
+S="${WORKDIR}/icon-${PV}"
+
+RDEPEND="
+	X? ( x11-libs/libX11:= )"
+DEPEND="
+	${RDEPEND}
+	X? (
+		x11-libs/libXpm
+		x11-libs/libXt
```

Comment:
Are these libraries not linked against => needed at runtime?

Apologies if I've asked this before.

Reply:
I'm not quite sure.. The source contains a stripped libXpm so the rdepend could simply be ensuring that the libXpm runtime prereqs are there.